app/controllers/placements/pages_controller.rb

```diff
@@ -1,3 +1,9 @@
 class Placements::PagesController < ApplicationController
   skip_before_action :authenticate_user!
+
+  def start
+    render locals: {
+      content: File.read(Rails.root.join("app/views/placements/pages/landing-page.md")),
```

Comment:
Sorry. I realise I [asked for this](https://github.com/DFE-Digital/itt-mentor-services/pull/597#discussion_r1592056755) – but maybe it actually makes sense to use an underscore in the filename instead of a dash?

```suggestion
      content: File.read(Rails.root.join("app/views/placements/pages/landing_page.md")),
```

So as to be consistent with the action name and `.html.erb` view file.